neurow/lib/neurow/broker/receiver_shard.ex

```diff
@@ -73,4 +77,9 @@ defmodule Neurow.Broker.ReceiverShard do
     :ets.delete_all_objects(table_1)
     {:reply, :ok, {table_0, table_1}}
   end
+
+  @impl true
+  def handle_call({:topic_count}, _from, {table_0, table_1}) do
```

Comment:
**Question** ❓ : how is this consumed / aggregate in a multi nodes setup ?

Reply:
Each node has its own copy of all topics. Once aggregated in a single dashboard, we will be able to check if all nodes have actually the same topic count.